main_menu/main_menu.gd

```diff
@@ -39,8 +39,7 @@ func _ready() -> void:
 	GlobalLobbyClient.lobby_joined.connect(_lobby_joined)
 	if not (OS.get_name() in ["Android", "iOS", "Web"]):
 		quit_button.show()
-	if not (OS.get_name() in ["Android", "iOS"]):
-		peer_name_line_edit.grab_focus()
+	peer_name_line_edit.grab_focus()
```

Comment:
The fourth batch have fixed and issue that will make grab_focus() doesn't enter edit mode, we should replace it with `LineEdit.edit()`